### PR TITLE
Add argparse

### PR DESCRIPTION
**Pull request by resnickmicah**
I wanted the option to pipe the words to a text-to-speech program, e.g. on MacOS: `python src/spell.py --print-letter=0 WTF | say`. This required adding the option to turn off the original letter in the output, and while I was there, I added the option to pick which alphabet dict you want to use.

### CODE REVIEW

File: src/spell.py
Comment (resnickmicah) [inline src/spell.py:31]: Black changed the formatting from single quotes to double-quotes, I can change that back if you want.

File: src/spell.py
Comment (resnickmicah) [inline src/spell.py:97]: This probably isn't necessary (or even reachable), I'm sure there's some way to return error messages if the value passed in isn't part of the `choices=["nato", "law_enforcement"]` param I added to the first argument. I'll clean this up later once I've had a closer look at the argparse docs.

**Comment by bondarevts** [comment]
Good idea, I wanted to introduce proper argument parsing at some point. Let's fix some small issues and integrate it in the main branch.

File: src/spell.py
Comment (bondarevts) [reply src/spell.py:31]: I guess it's fine. Let's stick to the black standard.

File: src/spell.py
Comment (bondarevts) [reply src/spell.py:97]: Let's keep it here to catch the case when a new alphabet is added.

**Comment by bondarevts** [comment]
I'll check the code tomorrow.

File: src/spell.py
Comment (bondarevts) [inline src/spell.py]: This version doesn't allow user to pass phrases, it only allows one word. I would like to keep the following usage possible:
```
$ spell a long phrase
```
Right now it fails with an error: "error: unrecognized arguments: long phrase"

File: src/spell.py
Comment (bondarevts) [inline src/spell.py]: I think we can turn this argument into a flag (e.g. "verbose"):
```
$ spell -v phrase
$ spell --verbose phrase
```

File: src/spell.py
Comment (bondarevts) [inline src/spell.py]: `law_enforcement` is too long. Let's change it to `police`.

**Comment by bondarevts** [comment]
Before merge we should update README.md as well to be consistent with the current implementation

**Review by bondarevts — CHANGES_REQUESTED** [review]
We should fix the following issues before merge:

- [x]  **support phrases with spaces**
- [x] turn `--print-letter` into a flag
- [x] update README.md

File: src/spell.py
Comment (resnickmicah) [reply src/spell.py]: Fixed!
```bash
$ ./src/spell.py spell a long phrase
Siérra
Papá
Écho
Líma
Líma
Álfa
Líma
Óscar
Novémber
Gólf
Papá
Hotél
Rómeo
Álfa
Siérra
Écho
```
I decided to remove spaces and other non-alpha characters from the output because it looked weird with verbose:
` : ` was the output line for any phrases with spaces, for example.
Let me know if you want me to leave spaces as part of the output.

Comment (bondarevts) [reply src/spell.py]: I would like to keep spaces, because it helps you to separate words visually. For verbose mode we can process spaces separately and print an empty line.  
Also I think that it's important to keep numbers as well, but I'm not sure how it should work. 

But I think these are issues for separate PRs. For now let's keep things simple and revert the behavior to the old one (if it works with your application).

File: src/spell.py
Comment (bondarevts) [inline src/spell.py]: The better way to write it would be 
```python
if not letter.lower() in spelling_map:
    continue
```

**Review by bondarevts — CHANGES_REQUESTED** [review]
So I think we can merge this PR if we 

- [x] update README to reflect the new interface and output 
- [x] remove the filter for non-alphabet characters.

Everything else is ready for merge!

**Review by bondarevts — APPROVED** [review]

**Pull request merged**
